pkgs/top-level/all-packages.nix

```diff
@@ -34516,6 +34516,8 @@ with pkgs;
 
   rep-gtk = callPackage ../development/libraries/rep-gtk { };
 
+  replxx = callPackage ../development/libraries/replxx { };
+
```

Comment:
Placing packages in `pkgs/by-name` is the new recommended way for new packages: https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md